sentinelhub/geopedia.py

```diff
@@ -27,27 +28,27 @@ def __init__(self, config=None):
         self.config = config or SHConfig()
         self._base_url = self.config.geopedia_rest_url
 
-    @staticmethod
-    def _parse_layer(layer, return_wms_name=False):
-        """ Helper function for parsing Geopedia layer name. If WMS name is required and wrong form is given it will
-        return a string with 'ttl' at the beginning. (WMS name can also start with something else, e.g. only 't'
-        instead 'ttl', therefore anything else is also allowed.) Otherwise it will parse it into a number.
-        """
-        if not isinstance(layer, (int, str)):
-            raise ValueError("Parameter 'layer' should be an integer or a string, but {} found".format(type(layer)))
 
-        if return_wms_name:
-            if isinstance(layer, int) or layer.isdigit():
-                return 'ttl{}'.format(layer)
-            return layer
+def _parse_geopedia_layer(layer, return_wms_name=False):
+    """ Helper function for parsing Geopedia layer name. If WMS name is required and wrong form is given it will
+    return a string with 'ttl' at the beginning. (WMS name can also start with something else, e.g. only 't'
+    instead 'ttl', therefore anything else is also allowed.) Otherwise it will parse it into a number.
+    """
+    if not isinstance(layer, (int, str)):
+        raise ValueError("Parameter 'layer' should be an integer or a string, but {} found".format(type(layer)))
+
+    if return_wms_name:
```

Comment:
But it is not just that. This function can either parse layer name into a layer id, e.g. `12345`, or it can parse it into a string identifier for WMS requests, e.g. `'ttl12345'`. And naming the parameter `return_as_string` does not reflect that.

Note that in this PR I just changed this from a method to a function. I didn't change its content.

Reply:
Hard to swallow, but I don't have better solution (apart perhaps from having two functions). Leave it as is.